Gems/ScriptCanvas/Code/Editor/Include/ScriptCanvas/Components/EditorScriptCanvasComponentSerializer.cpp

```diff
@@ -32,9 +32,7 @@ namespace AZ
         JsonSerializationResult::ResultCode result = BaseJsonSerializer::Load(outputValue
             , azrtti_typeid<AzToolsFramework::Components::EditorComponentBase>(), inputValue, context);
 
-        // load child data one by one...
-        result.Combine(BaseJsonSerializer::Load(outputValue, outputValueTypeId, inputValue, context));
```

Comment:
Question: what is the difference between this line and following actions?

Reply:
the following actions can perform some version checking. It could be loading an old component that used a asset reference, rather than a scripthandle